### PR TITLE
Tests added

### PR DESCRIPTION
**Pull request by AthulJoshi**
Tests for both routes has been added without auth.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,8 +33,10 @@
     "firebase-admin": "^9.4.2",
     "handlebars": "^4.7.6",
     "helmet": "^4.4.1",
+    "jest": "^26.6.3",
     "nodemailer": "^6.4.17",
     "pino": "^6.8.0",
+    "supertest": "^6.1.3",
```
Comment (AnandBaburajan) [inline package.json:39]: `jest` and `supertest` should be in devDependencies

**Comment by AnandBaburajan** [comment]
@AthulJoshi doubt: are actual mails sent to email IDs in `receiverIDs` during tests?
And the problem with CI was about not being able to access env vars right? I don't remember correctly.

**Comment by AthulJoshi** [comment]
The problem was Google rejecting sign in.

**Comment by AnandBaburajan** [comment]
> The problem was Google rejecting sign in. Yes, actual mails are sent.

Okay, let's keep the tests and we'll figure out the CI issues later. You can merge after fixing the dependencies comment above.
Then @iamAravindks can add auth. Everyone just needs to make sure that tests pass before pushing PRs (until CI is ready)

**Pull request merged**